crates/hdk/src/entry.rs

```diff
@@ -596,3 +608,19 @@ macro_rules! entry_def_index {
         }
     };
 }
+
+#[macro_export]
+macro_rules! entry_type {
```

Comment:
not directly relevant but it's hard to build headers for testing or validation/querying without it